swap_meet/item.py

```diff
@@ -1,2 +1,24 @@
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, category = '', condition = 0, age = 0):
+        self.category = category
+        self.condition = condition
+        self.age = age
+
+
+    def __str__(self):
+        return f"Hello World!"
+
+    def condition_description(self):
+        description = {
+            0: 'bad',
+            1: 'poor',
+            2: 'decent',
+            3: 'average',
+            4: 'good',
+            5: 'pristine'
+        }
+        
+        for rating, quality in description.items():
+            if self.condition == rating:
+                return quality 
```

Comment:
Excellent use of a data structure here to get the description based on condition - using a list enables you to avoid a long list of chained checks (if/elif/elif...) which can introduce bugs.

There are still edge case behaviors we'd need to consider and handle: what if the condition is outside the expected range? what if the condition is a non-integer value? Currently the function would return None if the condition on line 22 doesn't evaluate to True.

Also, when you have a dict, you can iterate through the keys without using .items like:
```python
for rating in description:
        if condition == rating:
            return description[rating]